src/EFCore.MySql/Internal/MySqlOptions.cs

```diff
@@ -16,6 +17,9 @@ public MySqlOptions()
         {
             ConnectionSettings = new MySqlConnectionSettings();
             ServerVersion = new ServerVersion(null);
+            CharSetAppendBehavior = CharSetAppendBehavior.AppendToAllAnsiColumns;
```

Comment:
Did you considered adding `AppendToUnicodeIndexAndKeyColumns` here? Does that make some of the functional tests fail?

Reply:
I think it's best to use the server default by default. If they have specified that it is ANSI, that should mean that Latin1 is safe. If they haven't specified anything, then it will default to Unicode. In that case I think it's best to use the server default, as users should pick a server default that is appropriate for the data they are storing.